build.gradle

```diff
@@ -41,6 +43,15 @@ checkstyle {
     checkstyleTest.enabled = false
 }
 
+task generateJava(type: Copy) {
```

Comment:
The template code generation is so we have access to the version to put into the wrapper.